### PR TITLE
Changelog 2.6

### PR DESCRIPTION
**Pull request by mauriciopoppe**
**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

/cc @jingxu97 @xing-yang

### CODE REVIEW

File: CHANGELOG/CHANGELOG-2.6.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# Changelog since v2.5.0
+
+## Changes by Kind
+
+### Feature
+
+- Add support to build arm/v7 images ([#139](https://github.com/kubernetes-csi/livenessprobe/pull/139), [@mauriciopoppe](https://github.com/mauriciopoppe))
+
+## Dependencies
```
Comment (msau42) [inline CHANGELOG/CHANGELOG-2.6.md:13]: Should we also update the dependencies in this repo?

Comment (mauriciopoppe) [reply CHANGELOG/CHANGELOG-2.6.md:13]: /hold
yes, holding until https://github.com/kubernetes-csi/livenessprobe/pull/141 is merged

**Comment by mauriciopoppe** [comment]
/remove-hold
#141 was merged, I regenerated the changelog file

**Pull request merged**